src/pages/PassportRedemption/RedemptionFooters.tsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import styled from 'styled-components';
 
 import { SubTitle, Button } from './style';
 
 export const NoRewardsFooter = () => {
   return (
-    <Footer>
+    <>
```

Comment:
is this needed?

Reply:
Do you mean the separate file for Footers? I'm not really sure. I can probably throw it all back into `RedemptionSelectScreen.tsx` if you're cool with that.